src/pages/admin/categories/CategoryList/index.js

```diff
@@ -1,12 +1,12 @@
 import React, { useState, useEffect } from 'react';
 import { Link, useHistory } from 'react-router-dom';
 import Card from 'react-bootstrap/Card';
-import Button from 'react-bootstrap/Button';
 import Dropdown from 'react-bootstrap/Dropdown';
 import { FaRegEdit } from 'react-icons/fa';
 import { RiDeleteBin2Fill } from 'react-icons/ri';
 import Pagination from '../../../../components/Pagination';
 import { VscFilter } from 'react-icons/vsc';
+import ButtonComponets from '../../../../components/Button';
```

Comment:
```suggestion
import Button from '../../../../components/Button';
```

will it throw an error if you use just button?